code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -1,4 +1,130 @@
 
+/******BRUTE******/
+
+/datum/chemical_reaction/trophazole
+	name = "Trophazole"
+	id  = /datum/reagent/medicine/trophazole
+	results = list(/datum/reagent/medicine/trophazole = 3)
+	required_reagents = list(/datum/reagent/copper = 1, /datum/reagent/acetone = 2,  /datum/reagent/phosphorus = 1, /datum/reagent/consumable/enzyme = 1)
```

Comment:
Oh boy, time to force the chef to hand over his bottle of enzyme (and then slide like 10u of it back to him so he can still do his job) every round as a CMO or chemist! That definitely won't get as repetitive as asking to be let into tech storage every round as a botanist or asking for secglasses every round as a detective!